RadarSDK/RadarAPIClient.m

```diff
@@ -337,60 +367,111 @@ - (void)searchGeofencesNear:(CLLocation * _Nonnull)near
 
     NSDictionary *headers = [RadarAPIClient headersWithPublishableKey:publishableKey];
 
-    [self.apiHelper requestWithMethod:@"GET" url:url headers:headers params:nil completionHandler:^(RadarStatus status, NSDictionary * _Nullable res) {
-        if (status != RadarStatusSuccess || !res) {
-            return completionHandler(status, nil, nil);
-        }
+    [self.apiHelper requestWithMethod:@"GET"
+                                  url:url
+                              headers:headers
+                               params:nil
+                    completionHandler:^(RadarStatus status, NSDictionary *_Nullable res) {
+                      if (status != RadarStatusSuccess || !res) {
+                          return completionHandler(status, nil, nil);
+                      }
+
+                      id geofencesObj = res[@"geofences"];
+                      NSArray<RadarGeofence *> *geofences = [RadarGeofence geofencesFromObject:geofencesObj];
+                      if (geofences) {
+                          return completionHandler(RadarStatusSuccess, res, geofences);
+                      }
+
+                      completionHandler(RadarStatusErrorServer, nil, nil);
+                    }];
+}
+
+- (void)searchPointsNear:(CLLocation *)near radius:(int)radius tags:(NSArray *)tags limit:(int)limit completionHandler:(RadarSearchPointsAPICompletionHandler)completionHandler
```

Comment:
tab down these params as we do with `searchGeofencesNear` above?

Reply:
must be some clang-format rule